hwilib/devices/digitalbitbox.py

```diff
@@ -575,8 +575,8 @@ def setup_device(self, label='', passphrase=''):
         to_send = {'seed': {'source': 'create', 'key': key, 'filename': backup_filename}}
         reply = send_encrypt(json.dumps(to_send).encode(), self.password, self.device)
         if 'error' in reply:
-            return {'success': False, 'error': reply['error']['message']}
-        return {'success': True}
+            raise DeviceFailureError(reply['error']['message'])
```

Comment:
I think this is a behavior change, throwing error instead of returning dictionary with error message?

Reply:
Yes. The error dict for the cli stuff is handled up the call chain. It is more pythonic to raise exceptions.